case-studies/wink.html

```diff
@@ -0,0 +1,125 @@
+---
+title: WINK Case Study
+layout: none
+class: gridPage
+cid: caseStudies
+---
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
+      <link rel="stylesheet" type="text/css" href="/css/style_wink.css">
+    <title>WINK Case Study</title>
+  </head>
+<body>
+
+
+<div class="banner1">
+  <h1>CASE STUDY: <img src="/images/wink_logo.png" class="logo" width="13%" style="margin-bottom:-4px"><br>
+    <div class="subhead">Cloud-Native Infrastructure Keeps Your Smart Home Connected</div>
+  </h1>
+</div>
+
```

Comment:
Do you want the top banner to have Kubernetes and its menu at the top? For example, https://kubernetes.io/case-studies/box/.